tsconfig.json

```diff
@@ -12,7 +12,8 @@
     "forceConsistentCasingInFileNames": true,
     "types": ["node", "jest", "@edge-runtime/types"],
     "noImplicitAny": false,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "experimentalDecorators": true
```

Comment:
[Apparently needed for decorators](https://www.typescriptlang.org/docs/handbook/decorators.html#introduction)